config.js

```diff
@@ -13,7 +13,7 @@ const config = {
     },
     live_preview: {
         enable: false,
-        host: 'api.contentstack.io'
+        host: 'rest-preview.contentstack.com'
```

Comment:
@harshithad0703 Please confirm the host. looks like we are adding some incorrect host here.

Reply:
@Deepak-Kharah , would you confirm if this host is correct?

Reply:
Yes, the host is correct

Reply:
Hi I double check the host. It's `rest-preview`